src/v/cluster/bootstrap_backend.cc

```diff
@@ -14,16 +14,19 @@
 #include "cluster/cluster_uuid.h"
 #include "cluster/commands.h"
 #include "cluster/logger.h"
+#include "cluster/members_manager.h"
 #include "cluster/types.h"
 #include "security/credential_store.h"
 
 namespace cluster {
 
 bootstrap_backend::bootstrap_backend(
   ss::sharded<security::credential_store>& credentials,
-  ss::sharded<storage::api>& storage)
+  ss::sharded<storage::api>& storage,
```

Comment:
nit: could you include the context about slow founders from the cover letter into the commit message here? It'll be much easier to find that way via `git blame`

Reply:
Makes sense but I don't feel it's the best place for that... however I can't come up with a better place